cmd/copy.go

```diff
@@ -995,6 +1006,7 @@ type cookedCopyCmdArgs struct {
 	// from arguments
 	source      common.ResourceString
 	destination common.ResourceString
+	isDfsDfs    bool //
```

Comment:
Minor: It would be appropriate to name this something similar to hns2hns instead of something which will be deprecated.